tests/everest/test_optimization_config.py

```diff
@@ -38,3 +38,26 @@ def test_optimization_config_options_unknown_options_error():
                 "merit_function el_bakry",
             ]
         )
+
+
+@pytest.mark.parametrize(
```

Comment:
I am assuming this also tests the sampler_config changes ?

Perhaps add an entry for ie: algorithm="A/B" expected ="A/B" and something that is expected to fail